k8s/namespaces/jenkins/patches/cftptl/cluster-00/jenkins.yaml

```diff
@@ -414,6 +414,7 @@ spec:
                           mkdir -pv /opt/jenkins
                           mount /dev/sdb1 /opt/jenkins
                           chown -R jenkinsssh:jenkinsssh /opt/jenkins
+                          set -e
```

Comment:
Any reason to not do this for the whole script?

Reply:
as we are seeing errors when doing a `umount /mnt/resource` _set -e_ would probably not allow us to actually check the main issue `cp /tmp/jenkinsssh_id_rsa /home/jenkinsssh/.ssh/id_rsa`

Reply:
Prob should get rid of that then